pages/public_cloud/containers_orchestration/managed_kubernetes/expose_your_services_using_a_load_balancer/guide.en-gb.md

```diff
@@ -95,20 +94,20 @@ spec:
     protocol: TCP
     targetPort: 80
   selector:
-    app: test-octavia
+    app: test-lb
   type: LoadBalancer
 ```
 6. Create a 'service' using the following command:
 ```shell
-kubectl create service test-lb-todel -f test-lb-todel.yaml
+kubectl apply -f test-lb-service.yaml
 ```
-7. Retrieve service IP address using the following command line:
+7. Retrieve Service IP address using the following command line:
 ```shell
-$ kubectl get service test-lb-todel
+$ kubectl get service test-lb-service -n=test-lb-ns
 NAME                 TYPE           CLUSTER-IP    EXTERNAL-IP      PORT(S)        AGE
-test-lb-todel   LoadBalancer   10.3.107.18   141.94.215.240   80:30172/TCP   12m
+test-lb-service      LoadBalancer   10.3.107.18   141.94.215.240   80:30172/TCP   12m
 ```
-8. Open a web browser and access: http://IP-retrieved-in-prevous-step
+8. Open a web browser and access: http://141.94.215.240
```

Comment:
warning to not display a running and working IP ;)

Reply:
yep, fixed